docker/package/tezos_setup_wizard.py

```diff
@@ -232,6 +232,35 @@ def check_blockchain_data(self):
             return False
         return True
 
+    # Check https://xtz-shots.io/base.json and collect the most recent snapshot
+    # that is suited for the chosen history mode and network
+    def get_snapshot_link(self):
+        self.config["snapshot_url"] = None
+        json_url = "https://xtz-shots.io/base.json"
```

Comment:
This was moved to a different URL
```suggestion
        json_url = "https://xtz-shots.io/tezos-snapshots.json"
```
I guess that the previous one is still accessible, but as we noticed it doesn't serve the same results.